NachoSpigot-Server/src/main/java/me/elier/nachospigot/config/NachoConfig.java

```diff
@@ -50,7 +50,7 @@ public static void init(File configFile) {
         File old_config = new File("nacho.json");
         if(old_config.exists()) migrate(old_config);
 
-        int configVersion = 2; // Update this every new configuration update
+        int configVersion = 3; // Update this every new configuration update
```

Comment:
```suggestion
        int configVersion = 4; // Update this every new configuration update
```
Config version updated in the #242